### PR TITLE
read request type (fetch vs ajax) for dependency telemetry

### PR DESCRIPTION
**Pull request by markwolff**
Related to https://github.com/Microsoft/ApplicationInsights-JS/pull/755

### CODE REVIEW

File: src/EnvelopeCreator.ts [hunk 1]
```diff
@@ -326,8 +326,9 @@ export class DependencyEnvelopeCreator extends EnvelopeCreator {
         let duration = bd.duration;
         let success = bd.success;
         let resultCode = bd.responseCode;
+        let requestType = bd.requestType; // fetch vs ajax
         let method = bd.properties ? bd.properties[HttpMethod] : "GET";
-        let baseData = new RemoteDependencyData(logger, id, absoluteUrl, command, duration, success, resultCode, method, customProperties, customMeasurements);
+        let baseData = new RemoteDependencyData(logger, id, absoluteUrl, command, duration, success, resultCode, method, requestType, customProperties, customMeasurements);
```
Comment (hectorhdzg) [inline src/EnvelopeCreator.ts]: Why the channel generate dependency events?

Comment (jpiyali) [reply src/EnvelopeCreator.ts]: Our channel downcasts types to CS 2.0. This is where its doing it.

**Comment by jpiyali** [comment]
no version update?

File: src/EnvelopeCreator.ts (hunk 1 above)
Comment (markwolff) [reply src/EnvelopeCreator.ts]: It is only generating the envelope. The event is generated in the dependencies plugin

**Comment by markwolff** [comment]
> no version update?

No sure what the state of versions are right now. Was going to wait for next release

**Comment by markwolff** [comment]
Closing for now. Will refactor in another PR

**Pull request closed without merging**